2_process/src/process_data.R

```diff
@@ -0,0 +1,25 @@
+############## Process data  ######################
+
+project_output_dir <- '1_process/out'
+
+dir.create(project_output_dir)
```

Comment:
time to take this line out of your file and instead lean on our team practice of creating directories manually (outside of the scripts that we save, commit, and/or use repeatedly)